api4/team_test.go

```diff
@@ -72,4 +72,6 @@ func TestCreateTeam(t *testing.T) {
 	th.LoginBasic()
 	_, resp = Client.CreateTeam(team)
 	CheckForbiddenStatus(t, resp)
+
+	TearDown()
```

Comment:
I don't think this will call teardown if the test fails. Could we defer teardown at the start instead?